h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -27,6 +28,8 @@
   public static final int HAS_HEADER = 1;
   public static final int GUESS_COL_CNT = -1;
 
+  public BytesStats bytesStats;
```

Comment:
What is the purpose of bytesStats? Is it just for the ARFF parser? If so we should probably have a derived ParseSetup for ARFF parser and put it there,  see AvroParseSetup